### PR TITLE
Added vidarr labels

### PR DESCRIPTION
**Pull request by pruzanov**

### CODE REVIEW

File: dellyGermline.wdl [hunk 1]
```diff
@@ -168,9 +169,15 @@ workflow dellyGermline {
       }
     ]
     output_meta: {
-      germlineSVs: "Filtered vcf file containing PASS structural variant calls",
-      germlineCNVs: "Filtered vcf file containing copy number variant calls"
+    germlineSVs: {
+        description: "Filtered vcf file containing PASS structural variant calls",
```
Comment (hannahdriver) [inline dellyGermline.wdl]: My mistake, this should not say "PASS" but rather: "Filtered vcf file containing structural variant calls"

File: README.md [hunk 2]
```diff
@@ -58,83 +58,83 @@ Parameter|Value|Default|Description
 
 ### Outputs
 
-Output | Type | Description
----|---|---
-`germlineSVs`|File|Filtered vcf file containing structural variant calls
-`germlineCNVs`|File|Filtered vcf file containing copy number variant calls
+Output | Type | Description | Labels
+---|---|---|---
+`germlineSVs`|File|Filtered vcf file containing PASS structural variant calls|vidarr_label: germlineSVs
```
Comment (hannahdriver) [inline README.md]: This should also be reverted to not include "PASS"

**Review by hannahdriver — APPROVED** [review]

File: dellyGermline.wdl (hunk 1 above)
Comment (pruzanov) [reply dellyGermline.wdl]: fixed

File: README.md (hunk 2 above)
Comment (pruzanov) [reply README.md]: Also fixed

File: README.md [hunk 3]
```diff
@@ -58,83 +58,83 @@ Parameter|Value|Default|Description
 
 ### Outputs
 
-Output | Type | Description
----|---|---
+Output | Type | Description | Labels
+---|---|---|---
 `germlineSVs`|File|Filtered vcf file containing structural variant calls
```
Comment (bibilujan) [inline README.md]: vidarr_label missing here

**Review by bibilujan — APPROVED** [review]

**Pull request merged**